tests/unit/multi.tcl

```diff
@@ -131,6 +131,22 @@ start_server {tags {"multi"}} {
         r exec
     } {} {cluster:skip}
 
+    test {EXEC fail on lazy expired WATCHed key} {
+        r flushall
+        r debug set-active-expire 0
+
+        r del key
+        r set key 1 px 2
+        r watch key
+
+        after 100
+
+        r multi
+        r incr key
+        r exec
+        r debug set-active-expire 1
+    } {}
```

Comment:
now we have a (silent) problem.
in tcl, the return value of the last statement is what's returned from a function.
at least i think that's the case (i never read the docs).
so in the past, the outcome of `r exec` was returning from the code snippet passed to the `test` proc.
and then the `test` proc was comparing it to what's in the last argument (the empty brackets).
now the return value of `r debug` is what's being returned.
we can either move the last `r debug` to outside the `test` (but that can cause issues since this test may need to get skipped on targets that do not support the DEBUG command.
or do this:
```suggestion
        assert_equal [r exec] {}
        r debug set-active-expire 1
    } {} {needs:debug}
```